### PR TITLE
Cypress 7.4.0

### PR DESCRIPTION
**Pull request by qmonmert**
Fix: https://github.com/jhipster/generator-jhipster/pull/14942 & https://github.com/jhipster/generator-jhipster/pull/15103

---

Please make sure the below checklist is followed for Pull Requests.

- [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [x] Tests are added where necessary
- [x] The JDL part is updated if necessary
- [x] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/cypress/templates/src/test/javascript/cypress/support/index.ts.ejs [hunk 1]
```diff
@@ -46,3 +46,10 @@ Cypress.Cookies.defaults({
 preserve: ['XSRF-TOKEN','SESSION', 'JSESSIONID']
 })
 <%_ } _%>
+
+// https://docs.cypress.io/api/events/catalog-of-events#Uncaught-Exceptions
+Cypress.on('uncaught:exception', (err, runnable) => {
```
Comment (mshima) [inline generators/cypress/templates/src/test/javascript/cypress/support/index.ts.ejs:51]: Don’t add to angular see https://github.com/jhipster/generator-jhipster/pull/14957/commits/be480fed73dc509179368ea7c86786eb0ed69b1d

Comment (qmonmert) [reply generators/cypress/templates/src/test/javascript/cypress/support/index.ts.ejs:51]: ok

Comment (mshima) [reply generators/cypress/templates/src/test/javascript/cypress/support/index.ts.ejs:52]: `clientFrameworkVue` is missing. The above commit adds it too.

Comment (qmonmert) [reply generators/cypress/templates/src/test/javascript/cypress/support/index.ts.ejs:52]: oups

**Comment by qmonmert** [comment]
@mshima do you think that we can merge?

**Comment by mshima** [comment]
Yes.
But this is a workaround not a real fix.

I was waiting to see if some react/vue developers were willing to fix the unhandled rejection before using the workaround.

I added the fix to bootstrap5 branch https://github.com/jhipster/generator-jhipster/commit/be480fed73dc509179368ea7c86786eb0ed69b1d by the time you created the initial fix for react.
Bootstrap5 was not scrolling correctly with cypress 6.

Since a new release is planned let's merge.

**Comment by qmonmert** [comment]
ok

**Comment by qmonmert** [comment]
@pascalgrimaud https://opencollective.com/generator-jhipster/expenses/41627 :)

**Comment by pascalgrimaud** [comment]
@qmonmert : approved, thanks for your work

**Pull request merged**